internal/pkg/service/stream/storage/level/local/encoding/pipeline_test.go

```diff
@@ -175,6 +177,8 @@ func TestEncodingPipeline_Sync_Wait_ToDisk(t *testing.T) {
 		tc.Logger.Infof(ctx, "TEST: write unblocked")
 	}()
 	tc.ExpectWritesCount(t, 2)
+	// Pri triger sync sa vytvori novy notifier, stary sa uzavrie a odblokuje 2 writy.
+	// Novy notifier sa musi predat do encoderu aby sa zablokoval dalsi zapis
```

Comment:
English please.

Reply:
ah sorry, did not checked files before. My fault